### PR TITLE
change passphrase confirmation logic

### PR DESCRIPTION
**Pull request by sophoah**
- modify getPassphrase logic to ask for confirmed passphrase (two input) based on the argument passphraseconfirmation
- update all the occurrence of getPassphrase, now the below need confirmation (double input of passphrase)
  - ONE account creation
  - BLS account creation
  - generate BLS Key
  - recover from mnemonic since the current method set a passphrase
  - save BLS Key
- and those don't need confirmation (single input of passphrase) :
  - importKS
  - importPK
  - exportKS
  - exportPK
  - recover BLS Key
  - create validator
  - edit validator
  - delegation
  - undelegation
  - collect reward
  - bulk transfer
  - normal transfer

PS : not sure why my commit from 11 March are also shown here.

### CODE REVIEW

**Comment by sophoah** [comment]
@fxfactorial @Daniel-VDM appreciate your review here :) Thanks

**Comment by sophoah** [comment]
forgot to attach the tests done
```bash
root@982ad5aa0ec1:~/go/src/github.com/harmony-one/go-sdk# make
source /root/go/src/github.com/harmony-one/harmony/scripts/setup_bls_build_flags.sh && GO111MODULE=on go build -o ./dist/hmy -ldflags="-X main.version=v326 -X main.commit=a6deafe-dirty -X main.builtAt=2020-03-15T11:12:47+0000 -X main.builtBy=@harmony.one" cmd/main.go
cp ./dist/hmy hmy
root@982ad5aa0ec1:~/go/src/github.com/harmony-one/go-sdk# ./hmy keys recover-bls-key $(pwd)/d51f4ec87fe048ae1ed3a22990f9ff0387ad64dd27cb353bb4254746ae9aaa3d863ce0f6bdb493a4b676bef22db74f10.key --passphrase
Enter passphrase:
{
  "public-key": "0xd51f4ec87fe048ae1ed3a22990f9ff0387ad64dd27cb353bb4254746ae9aaa3d863ce0f6bdb493a4b676bef22db74f10",
  "private-key": "0xe0adbe56c00325d930f9f7dde7ca1b6c5372d388ae10a0251379916383e6e22d"
}

root@982ad5aa0ec1:~/go/src/github.com/harmony-one/go-sdk# ./hmy keys generate-bls-key
{
  "public-key": "0c90320e63cca0ddf51ecd9932896eb0d3b3cb108b1c1b5a2f67f184395329bc1c9e08d4285f53cb1c16f92412cf5f0d",
  "private-key": "76dd97f5f754b88748c4f22018bc92c90c24275ca921ba8d283663680aa3ad3f",
  "encrypted-private-key-path": "/root/go/src/github.com/harmony-one/go-sdk/0c90320e63cca0ddf51ecd9932896eb0d3b3cb108b1c1b5a2f67f184395329bc1c9e08d4285f53cb1c16f92412cf5f0d.key"
}
root@982ad5aa0ec1:~/go/src/github.com/harmony-one/go-sdk# ./hmy keys generate-bls-key --passphrase
Enter passphrase:
Repeat the passphrase:

{
  "public-key": "a0aa6017e59f933831d4e9d80957db38639e3d21762e4e215f91a4112bcbf27fdcb8976ddbd022945f47fe0a735a0e84",
  "private-key": "5e0b7f4959f0e7ce42e08dbd0aba144dd38e3804073713492c8dee7221059460",
  "encrypted-private-key-path": "/root/go/src/github.com/harmony-one/go-sdk/a0aa6017e59f933831d4e9d80957db38639e3d21762e4e215f91a4112bcbf27fdcb8976ddbd022945f47fe0a735a0e84.key"
}
root@982ad5aa0ec1:~/go/src/github.com/harmony-one/go-sdk# ./hmy keys recover-bls-key $(pwd)/a0aa6017e59f933831d4e9d80957db38639e3d21762e4e215f91a4112bcbf27fdcb8976ddbd022945f47fe0a735a0e84.key --passphrase
Enter passphrase:
{
  "public-key": "0xa0aa6017e59f933831d4e9d80957db38639e3d21762e4e215f91a4112bcbf27fdcb8976ddbd022945f47fe0a735a0e84",
  "private-key": "0x5e0b7f4959f0e7ce42e08dbd0aba144dd38e3804073713492c8dee7221059460"
}
root@982ad5aa0ec1:~/go/src/github.com/harmony-one/go-sdk# ./hmy keys recover-bls-key $(pwd)/d51f4ec87fe048ae1ed3a22990f9ff0387ad64dd27cb353bb4254746ae9aaa3d863ce0f6bdb493a4b676bef22db74f10.key --passphrase-file blspass.txt
{
  "public-key": "0xd51f4ec87fe048ae1ed3a22990f9ff0387ad64dd27cb353bb4254746ae9aaa3d863ce0f6bdb493a4b676bef22db74f10",
  "private-key": "0xe0adbe56c00325d930f9f7dde7ca1b6c5372d388ae10a0251379916383e6e22d"
}
root@982ad5aa0ec1:~/go/src/github.com/harmony-one/go-sdk#
```

File: cmd/subcommands/keys.go [hunk 1]
```diff
@@ -41,7 +41,8 @@ var (
 // getPassphrase fetches the correct passphrase depending on if a file is available to
 // read from or if the user wants to enter in their own passphrase. Otherwise, just use
 // the default passphrase.
-func getPassphrase() (string, error) {
+// argument passphraseconfirmation is true when confirmation is required false otherwise 
+func getPassphrase(passphraseconfirmation bool) (string, error) {
```
Comment (fxfactorial) [inline cmd/subcommands/keys.go]: Don't make a function with a bool parameter. instead make two functions that do what you want, that is, `getPassphraseWithConfirm`, `getPassphrase` (assume no confirm on that one)

Comment (sophoah) [reply cmd/subcommands/keys.go]: @fxfactorial just made the change as suggested. Thanks

**Pull request merged**